### DISCUSSION TITLE
Extending the HTML head tag

### DISCUSSION THREAD

**Discussion by robigan** [jpanther/lynx#22] [Q&A]
Heyo, is it possible to extend the head tag with my own link/meta tags? If not, could it be done by loading something like a partial or shortcode? Thanks

**Comment by jpanther** [comment]
Yes, it's possible to extend the head section of the HTML. Simply include an `extend-head.html` file in the `partials` directory of your project and it will be included just before the `</head>` of the Hugo output.
